k8s/commander.sh

```diff
@@ -2,8 +2,8 @@
 set -exo pipefail
 GREEN='\033[1;32m'
 NC='\033[0m' # No Color
-SLACK_CHANNEL=sumodev
-DOCKER_HUB="https://hub.docker.com/r/mozmeao/kitsune/tags/"
+SLACK_CHANNEL=itsre-sumo
```

Comment:
Shall we use the existing channel? It's already well known by users and it usually doesn't have any noise apart from dev related topics. The new channel has also logs from `yar` which will make it harder to locate the builds.